internals/generators/container/index.js

```diff
@@ -47,10 +47,22 @@ module.exports = {
   }],
   actions: (data) => {
     // Generate index.js and index.test.js
+    let componentTemplate;
```

Comment:
The build is failing using node 5 (we support the last 3 versions, so 5, 6, 7). Might be best to use `var` here instead